docs/src/prototypes/chatMessages/ControlMessages/GroupControlMessages.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import * as keyboardKey from 'keyboard-key'
+
+import { List, chatBehavior, Accessibility, Flex, Icon } from '@stardust-ui/react'
+import ControlMessage from './ControlMessage'
+
+const overridenChatBehavior: Accessibility<any> = props => {
+  const behaviorData = chatBehavior(props)
+
+  behaviorData.attributes.root = {
+    ...behaviorData.attributes.root,
+    'data-is-focusable': true,
+  }
+
+  behaviorData.focusZone = {
+    mode: behaviorData.focusZone.mode,
+    props: {
+      ...behaviorData.focusZone.props,
+      shouldFocusOnMount: true,
+      shouldFocusInnerElementWhenReceivedFocus: true,
+      defaultTabbableElement: undefined,
+    },
+  }
+  return behaviorData
+}
+
+const GroupControlMessages = () => {
```

Comment:
please consider using `createComponent` in this case as well - it might help to standardize styling. Also, it migh allow you to move keyboard handling to the behavior

Reply:
we won't be able to move keyboard handling, as `actionHandlers` are not yet supported in `createComponent` thus we can't move it to behaviors. So `createComponent` might be useful only for styles 🤷‍♀ @mnajdova what do you think?